Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 def dirPrefix = 'reframe-ci'
 def loginBash = '#!/bin/bash -l'
 def bashScript = 'ci-scripts/ci-runner.bash'
-def machinesList = ['daint', 'dom', 'kesch']
+def machinesList = params.machines.split()
```

Comment:
Where these `params` are gotten from? From the Jenkins project? Could you then remove `Kesch` from there?

Reply:
The `params` is injected by Jenkins and if the build is not triggered manually the parameters have the default values.

Reply:
So the defaults are set by the Jenkins project?

Reply:
You add a parameter and give it default values yes.